runtime/runtime.go

```diff
@@ -352,6 +358,14 @@ func (rt *Runtime) Serve(ctx context.Context) error {
 	rt.server.Handler = NewLoggingHandler(rt.server.Handler)
 	rt.server.DiagnosticHandler = NewLoggingHandler(rt.server.DiagnosticHandler)
 
+	if rt.Params.ReadyTimeout > 0 {
+		logrus.Debugf("Waiting for plugins activation (%v).", rt.Params.ReadyTimeout)
+		if err := util.WaitFunc(rt.Manager.PluginsReady, rt.Params.ReadyTimeout, 100*time.Millisecond); err != nil {
```

Comment:
I think the params are swapped for this. I noticed while testing it looks like it will always stop the server after 100ms instead of respecting the provided duration. Ex:

```
{15:10} ✘ ~/p/g/s/g/o/opa:master ✓ ❯ go run . run -s --set services.foo.url="http://some-bad-url" --set bundles.foo-bundle.service=foo -l debug --ready-timeout 60s
{"addrs":[":8181"],"diagnostic-addrs":[],"insecure_addr":"","level":"info","msg":"Initializing server.","time":"2020-08-05T15:10:12-07:00"}
{"level":"info","msg":"Starting bundle downloader.","name":"foo-bundle","plugin":"bundle","time":"2020-08-05T15:10:12-07:00"}
{"level":"debug","msg":"Download starting.","time":"2020-08-05T15:10:12-07:00"}
{"headers":{"User-Agent":["Open Policy Agent/ (darwin, amd64)"]},"level":"debug","method":"GET","msg":"Sending request.","time":"2020-08-05T15:10:12-07:00","url":"http://some-bad-url/bundles/foo-bundle"}
{"level":"debug","msg":"Waiting for plugins activation (1m0s).","time":"2020-08-05T15:10:12-07:00"}
{"headers":{"Content-Type":["application/json"],"User-Agent":["Open Policy Agent/ (darwin, amd64)"]},"level":"debug","method":"POST","msg":"Sending request.","time":"2020-08-05T15:10:12-07:00","url":"https://telemetry.openpolicyagent.org/v1/version"}
{"level":"error","msg":"Bundle download failed: request failed: Get \"http://some-bad-url/bundles/foo-bundle\": dial tcp: lookup some-bad-url: no such host","name":"foo-bundle","plugin":"bundle","time":"2020-08-05T15:10:12-07:00"}
{"level":"debug","msg":"Waiting 0s before next download/retry.","time":"2020-08-05T15:10:12-07:00"}
{"level":"debug","msg":"Download starting.","time":"2020-08-05T15:10:12-07:00"}
{"headers":{"User-Agent":["Open Policy Agent/ (darwin, amd64)"]},"level":"debug","method":"GET","msg":"Sending request.","time":"2020-08-05T15:10:12-07:00","url":"http://some-bad-url/bundles/foo-bundle"}
{"level":"error","msg":"Bundle download failed: request failed: Get \"http://some-bad-url/bundles/foo-bundle\": dial tcp: lookup some-bad-url: no such host","name":"foo-bundle","plugin":"bundle","time":"2020-08-05T15:10:12-07:00"}
{"level":"debug","msg":"Waiting 183.194692ms before next download/retry.","time":"2020-08-05T15:10:12-07:00"}
{"err":"timeout","level":"error","msg":"Failed to wait for plugins activation.","time":"2020-08-05T15:10:12-07:00"}
{"level":"debug","msg":"Download starting.","time":"2020-08-05T15:10:12-07:00"}
{"headers":{"User-Agent":["Open Policy Agent/ (darwin, amd64)"]},"level":"debug","method":"GET","msg":"Sending request.","time":"2020-08-05T15:10:12-07:00","url":"http://some-bad-url/bundles/foo-bundle"}
{"level":"error","msg":"Bundle download failed: request failed: Get \"http://some-bad-url/bundles/foo-bundle\": dial tcp: lookup some-bad-url: no such host","name":"foo-bundle","plugin":"bundle","time":"2020-08-05T15:10:12-07:00"}
{"level":"debug","msg":"Waiting 258.355775ms before next download/retry.","time":"2020-08-05T15:10:12-07:00"}
{"headers":{"Connection":["keep-alive"],"Content-Length":["59"],"Content-Type":["application/json"],"Date":["Wed, 05 Aug 2020 22:10:12 GMT"]},"level":"debug","method":"POST","msg":"Received response.","status":"400 Bad Request","time":"2020-08-05T15:10:12-07:00","url":"https://telemetry.openpolicyagent.org/v1/version"}
{"err":"server replied with HTTP 400","level":"debug","msg":"Unable to send OPA version report.","time":"2020-08-05T15:10:12-07:00"}
{"level":"info","msg":"Stopping bundle downloader.","name":"foo-bundle","plugin":"bundle","time":"2020-08-05T15:10:12-07:00"}
exit status 1
```

We should fix that, and I guess see what we can do about getting test coverage for this code.